.github/workflows/build-dev-infra.yml

```diff
@@ -61,9 +61,11 @@ jobs:
               --parameters deployFrontendCosmos="true" \
               --parameters persist="true"
 
-#            # management cluster
-#            az deployment group create \
-#              --name "mgmt-cluster-${GITHUB_RUN_ID}" \
-#              --resource-group "${RESOURCEGROUP}" \
-#              --template-file templates/mgmt-cluster.bicep \
-#              --parameters configurations/mgmt-cluster.bicepparam \
+            SVC_CLUSTER_NAME=$(az deployment group show --resource-group "${RESOURCEGROUP}" --name "svc-cluster-${GITHUB_RUN_ID}" --output tsv --query properties.outputs.aksClusterName.value)
+            # service cluster rolebindings
+            az deployment group create \
+              --name "svc-rolebindings-${GITHUB_RUN_ID}" \
+              --resource-group "${RESOURCEGROUP}" \
+              --template-file templates/dev-aks-roleassignments.bicep \
+              --parameters aksClusterName=${SVC_CLUSTER_NAME} \
+              --parameters location="eastus"
```

Comment:
can we pull location into a var used for GH actions, or set it higher up?  

It looks like we're using it in a couple places and we may want to change this at some point to deploy to a region that doesn't have 3 azs.

Reply:
everywhere else we define `location` based on the resource group location
`param location string = resourceGroup().location`

updated the bicep template to do the same so that the `--parameters location="eastus"` flag could be removed and location will always be based on the resource group, allowing for us to target other regions